### PR TITLE
fix: minor project creation flow polish

### PR DESCRIPTION
**Pull request by piperchester**
This is some very minor polish I wanted to contribute when working on #669. I simplified the title flow, result:

Screen Shot 2021-07-25 at 11 43 36 AM

Also added a `CODEOWNERS` file so we can start scoping / tagging areas of the project for context and review.

### CODE REVIEW

File: frontend/public/texts/project_texts.js [hunk 1]
```diff
@@ -773,7 +773,7 @@ export default function getProjectTexts({ project, user, url_slug, locale }) {
       ),
     },
     title_with_explanation_and_example: {
-      en: "Title (Use a short, english title, e.g. 'Generating energy from ocean waves')",
+      en: "Title (Use a short, compelling title, e.g. 'Generating energy from ocean waves')",
```
Comment (piperchester) [inline frontend/public/texts/project_texts.js:776]: Mentioning "English" twice seemed redundant -- so I added "compelling" and removed the InfoIcon bit that was under the form. This simplifies this screen and improves the UI IMO. 

Screen Shot 2021-07-25 at 11 43 36 AM

File: frontend/src/components/shareProject/ShareProject.js [hunk 2]
```diff
@@ -107,14 +113,6 @@ export default function Share({
...
       key: "name",
       value: project.name,
-      bottomLink: (
-        <Typography className={classes.BottomLinkFlex}>
-          <InfoOutlinedIcon className={classes.infoIcon} />
-          <Typography component="span">
-            {texts.use_a_title_that_makes_people_curious_to_learn_more_about_your_project}
-          </Typography>
-        </Typography>
-      ),
```
Comment (piperchester) [inline frontend/src/components/shareProject/ShareProject.js:117]: Removed this (see outer screenshot of the pull request).

File: frontend/src/components/shareProject/ShareProject.js [hunk 3]
```diff
@@ -170,11 +168,15 @@ export default function Share({
       {locale === "en" && (
         <div className={classes.appealBox}>
           <Typography color="secondary" className={classes.appealText}>
-            Please make sure to only use English when sharing a project.
+            Please make sure to{" "}
+            <Typography component="span" className={classes.bold}>
+              only use English when sharing a project
+            </Typography>
```
Comment (piperchester) [inline frontend/src/components/shareProject/ShareProject.js:174]: Wanted to emphasize the bold bit, as opposed to bolding everyhing.

**Review by ddhanesha — APPROVED** [review]
This looks good to me. Thanks for adding CODEOWNERS section. 👍🏽 
Approved!

**Pull request merged**